tmt/frameworks/beakerlib.py

```diff
@@ -36,7 +36,16 @@ def get_environment_variables(
         return Environment({
             'BEAKERLIB_DIR': EnvVarValue(invocation.path),
             'BEAKERLIB_COMMAND_SUBMIT_LOG': EnvVarValue(
-                f'bash {tmt.steps.execute.TMT_FILE_SUBMIT_SCRIPT.path}')
+                f'bash {tmt.steps.execute.TMT_FILE_SUBMIT_SCRIPT.path}'),
+
+            # The command in this variable gets called with every `rlPhaseEnd` call in beakerlib.
+            'BEAKERLIB_COMMAND_REPORT_RESULT': EnvVarValue(
+                f'bash {tmt.steps.execute.TMT_REPORT_RESULT_SCRIPT.path}'),
```

Comment:
Should we use `tmt.utils.DEFAULT_SHELL` and maybe even `tmt.utils.SHELL_OPTIONS`? Or, isn't the `TMT_REPORT_RESULT_SCRIPT.path` already executable, and does not need any `bash` at all?

Reply:
Thanks for the hint. Just `tmt.steps.execute.TMT_REPORT_RESULT_SCRIPT.path` seems to be working fine.

Reply:
I guess the same applies for the `tmt-file-submit` script, proposing in #3310.